hoodie-spark/src/main/java/com/uber/hoodie/HoodieDataSourceHelpers.java

```diff
@@ -74,4 +78,14 @@ public static HoodieTimeline allCompletedCommitsCompactions(FileSystem fs, Strin
       return metaClient.getCommitTimeline().filterCompletedInstants();
     }
   }
+
+  public static void initializeNewDataset(String tableName, String storageType, Configuration conf,
```

Comment:
There is a similar method HoodieTableMetaClient.initTableType. If it is not a perfect fit, can we keep the methods in same class ? Thoughts ?

Reply:
the `HOODIE_ARCHIVELOG_FOLDER_PROP_NAME` setting is what makes this a bit specific to these modules.Will work on consolidating methods there.